frame/beefy-mmr/src/mock.rs

```diff
@@ -77,16 +74,16 @@ impl frame_system::Config for Test {
 	type Lookup = IdentityLookup<Self::AccountId>;
 	type Header = Header;
 	type Event = Event;
-	type BlockHashCount = BlockHashCount;
+	type BlockHashCount = ConstU64<250>;
 	type Version = ();
 	type PalletInfo = PalletInfo;
 	type AccountData = ();
 	type OnNewAccount = ();
 	type OnKilledAccount = ();
 	type SystemWeightInfo = ();
-	type SS58Prefix = SS58Prefix;
+	type SS58Prefix = ConstU16<42>;
```

Comment:
This is `u8` not `u16`.

Reply:
<img width="1440" alt="Screenshot 2021-12-15 at 11 12 17" src="https://user-images.githubusercontent.com/11888724/146167712-c29d9a83-01a7-4457-b9d4-5fad814c9273.png">

When i use ConstU8 i get an error when testing, this screenshot i attached shows that the type SS8Prefix is a Get<U16> basically

Reply:
that's right, resolving this